crates/lib/src/watchexec.rs

```diff
@@ -105,11 +105,12 @@ impl Watchexec {
 			let fs = SubTask::spawn("fs", fs::worker(fs_r, er_s.clone(), ev_s.clone()));
 			let signal =
 				SubTask::spawn("signal", signal::source::worker(er_s.clone(), ev_s.clone()));
+			let keyboard = SubTask::spawn("keyboard", keyboard::worker(er_s.clone(), ev_s.clone()));
```

Comment:
see how the fs worker has three arguments and the first is a watch channel of the fs working data? that's how you get the config in there